src/main/scala/org/broadinstitute/dsde/firecloud/service/UserService.scala

```diff
@@ -40,6 +40,9 @@ object UserService {
   val billingPath = FireCloudConfig.Rawls.authPrefix + "/user/billing"
   val billingUrl = FireCloudConfig.Rawls.baseUrl + billingPath
 
+  val refreshTokenDatePath = FireCloudConfig.Rawls.authPrefix + "/user/refreshTokenDate"
+  val refreshTokenDateUrl = FireCloudConfig.Rawls.baseUrl + refreshTokenDatePath
+
```

Comment:
OAuthService already defines these (OAuthService.remoteTokenDatePath) ... can you centralize, or point one at the other?